frank/default_parameters.json

```diff
@@ -11,7 +12,10 @@
 
   "modify_data" : {
     "cut_data"         : false,
-    "cut_baseline"     : null
+    "cut_range"        : null,
+    "correct_weights"  : false,
+    "norm_by_wle"      : false,
+    "wle"              : null
```

Comment:
I don't like this at all the wle should be provided in the uv table and we should detect automatically from the uv table that units to use

Reply:
The wle is not always printed in the table header when extracting a table in CASA (e.g., if something other than uvplot is used to extract the table).

Reply:
I think we should just not handle the case where baseline is in meters

Reply:
Either that, or provide a specification for a file format that frankenstein understands and that users must use.

Reply:
I still think this needs addressing.